compiler/parser/RdrHsSyn.hs

```diff
@@ -2985,11 +2999,11 @@ checkTemplateDeclConstraints nloc ens agr bns
 mkTemplateDecl
   :: Located RdrName -- The template name
   -> LHsType GhcPs -- The template's record type
-  -> Located [Located TemplateBodyDecl]  -- Functions and choice groups
+  -> Located [Located TemplateBodyDecl]  -- Functions, controlled choice groups, flexible choices
   -> P (OrdList (LHsDecl GhcPs)) -- Desugared declarations
 mkTemplateDecl lname@(L nloc _name) fields (L _ decls) = do
   let dataName = L nloc (HsTyVar noExt NotPromoted lname)
-      (ens, sig, obs, agr, cgs, binds) = extractTemplateBodyDecls decls
+      (ens, sig, obs, agr, cgs, binds, _flxs) = extractTemplateBodyDecls decls
```

Comment:
I suppose we ignore `_flxs` because we're only doing the parser here but not the desugaring.

Reply:
That's right. I'll get this in and follow up on desugaring in a later PR.